testutils/src/mock.ts

```diff
@@ -826,7 +832,7 @@ namespace Private {
   export function changeKernel(
     kernel: Kernel.IKernelConnection,
     partialModel: Partial<Kernel.IModel>
-  ): Promise<Kernel.IModel> {
+  ): Promise<Kernel.IKernelConnection> {
```

Comment:
All happy paths return one of the elements in `RUNNING_KERNELS`, and that array is of type `IKernelConnection`. Fixing return type resolves several type errors above.

Reply:
:+1: